README.md

```diff
@@ -1,14 +1,31 @@
-NYPLBase
+# NYPLBase #
 ========
```

Comment:
I don't think you need this separator under NYPLBase. The heading is big enough and left space until the next paragraph starts.

Reply:
Cool! Updated!